### PR TITLE
faet: Update README.md

### PR DESCRIPTION
**Pull request by santai-wang-lb**
大家幫我看一下有沒有什麼東西可以加的

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,10 +29,45 @@ Our idea is build a telegram bot for community management, it has 3 functionalit
...
+### 5. Put appScript into Google appScipt endpoint
+1. Create 4 new project in google appScript
+2. Paste the 4 `.gs` files individually into each project
+#### Reference
+[Google Apps Script](https://developers.google.com/apps-script)
+[Google appScript Console](https://script.google.com/home/start)
 
 ## Reference
 
 [Idea Agenda](https://docs.google.com/presentation/d/1QEUo_2glJO5DsWRdGZ5p6H0i_8olk6lu865NMJp1fm0/edit#slide=id.g2052594a4f0_2_63)
```
Comment (lcgcharles) [inline README.md:73]: Repeat. Same link for **Demo slide**

Comment (santai-wang-lb) [reply README.md:203]: Removed

Comment (santai-wang-lb) [reply README.md]: Removed, plz resolve it

Comment (santai-wang-lb) [reply README.md]: https://github.com/LeadBest/ton-hackthon/tree/README_update

**Pull request merged**